hc-tree-view-behavior.html

```diff
@@ -261,28 +261,29 @@
      * @private
      * @param {Array.<Object>} items
      * @param {String} state
-     * @param {Object} [parts = this._parts]
      */
-    _updateState(items, state, parts = this._parts) {
-      if (!items || !parts) return;
-      // Filter items to remove children that have a parent in the array
-      items = items.filter((item) => {
-        item.state = state;
-        return !items.some((i) => i[this.itemIdName] === item[this.itemParentIdName]);
-      });
-      // Pass the filtered items into _updateChildren
-      items.forEach((item) => {
-        // Find the item in the parts object
-        const parentId = this.list._getParentId(item);
-        const partItem = parts[parentId].find((i) => {
-          return i[this.itemIdName] === item[this.itemIdName];
+    _updateState(items, state) {
```

Comment:
What is the benefit of removing `parts` from the params list?

Reply:
It's captured on call if it's here and within the debounce it will used the on-call captured version instead of the latest that has been updated, this was added to force into a micro task so it can finish it's other operations before continuing, since _parts wasn't computed by the time this was called.